website/content/en/wgs/platforms/_index.md

```diff
@@ -18,10 +20,12 @@ or the meetings described below.
 
 ## Meetings
 
+* Please RSVP for meetings on the [TAG App Delivery Community Groups Page](https://community.cncf.io/tag-app-delivery/)
 * Meeting schedule: 2nd and 4th Tuesday of each month at [1600 Europe/London time](https://www.timeanddate.com/worldclock/converter.html?iso=20240514T150000&p1=136)
     * [2nd Tuesday event](https://www.google.com/calendar/event?eid=cWE0ZHMyMThicDF1dDZsa3N2ZGlqYmt2Z2pfMjAyNDA1MTRUMTUwMDAwWiBsaW51eGZvdW5kYXRpb24ub3JnX281YXZqbHZ0MmNhZTlicTdhOTVlbWM0NzQwQGc&ctz=UTC)
```

Comment:
This is set to a date in May, I think it won't update moving forward. Plus it has UTC at the end of the URL which makes me nervous as we are now pinned to British time 👀

Reply:
I pointed out the issue with the calendar in a separate slack thread with Lian. Maybe we should just remove that link and point them to the tag app delivery events page?

Reply:
Yeah that might make more sense tbh. Saves us from having to know all these instances every time we change a time.

Reply:
@abangser @lianmakesthings how does it look now?